python_modules/dagster-graphql/dagster_graphql/schema/instigation.py

```diff
@@ -277,6 +278,9 @@ def __init__(self, _, tick):
     def resolve_id(self, _):
         return "%s:%s" % (self._tick.instigator_origin_id, self._tick.timestamp)
 
+    def resolve_evaluationId(self, _: ResolveInfo) -> str:
+        return str(self._tick.tick_id)
```

Comment:
sadly, because evaluation IDs pre-date using ticks for AMP, the evaluation ID is not the same as the tick ID. I think what you are looking for here is

self._tick.tick_data.auto_materialize_evaluation_id

If the intention is something that will be consistent between sensors,schedules,and AMP, this is sadly one place where they are currently not consistent